backend/src/main/java/com/eyetrade/backend/service/CurrencyConverterService.java

```diff
@@ -40,7 +42,8 @@ public Double findRate(CurrencyType type, CurrencyRecord record) {
                 return record.getSterlingRate();
             case USD:
                 return record.getDollarRate();
+            default:
+                throw new RuntimeException(NO_SUCH_CURRENCY_TYPES);
```

Comment:
I think we don't need that default case because if the currency type is not proper spring will return badrequest since the input type in controller is enum